tests/test_smartredis.py

```diff
@@ -93,14 +93,12 @@ def test_exchange(fileutils, wlmutils):
 
     # get and confirm statuses
     statuses = exp.get_status(ensemble)
-    if not all([stat == status.STATUS_COMPLETED for stat in statuses]):
+    try:
+        assert all([stat == status.STATUS_COMPLETED for stat in statuses])
+    finally:
+        # stop the orchestrator
         exp.stop(orc)
-        assert False  # client ensemble failed
-
-    # stop the orchestrator
-    exp.stop(orc)
-
-    print(exp.summary())
+        print(exp.summary())
```

Comment:
Delete this or maybe if you want it to stick around if the test fails, dump it to a file in the `test_output`

Reply:
dropped at [73b0df7](https://github.com/CrayLabs/SmartSim/pull/379/commits/73b0df74d3c8239cbe99f8d07fd59304f287b34c)